README.md

```diff
@@ -1,8 +1,8 @@
 # <img src="src/docs/images/roxana-logo.png">
 
-A [Spring projects](https://spring.io/projects) based framework created to guarantee a standardized RESTful services responses.
+A [Spring](https://spring.io/projects) based project framework created to guarantee a standardized RESTful services responses.
```

Comment:
Eu acredito que ficaria melhor "A Spring based framework, created to ...", por que o que eu quis dizer é "Um framework baseado em String, criado para ...".

Reply:
:+1: Vou ajustar